homeassistant/components/fitbit/sensor.py

```diff
@@ -1,174 +1,85 @@
-    "activities/tracker/activityCalories": ["Tracker Activity Calories", "cal", "fire"],
-    "activities/tracker/calories": ["Tracker Calories", "cal", "fire"],
-    "activities/tracker/distance": ["Tracker Distance", "", "map-marker"],
-    "activities/tracker/elevation": ["Tracker Elevation", "", "walk"],
-    "activities/tracker/floors": ["Tracker Floors", "floors", "walk"],
-    "activities/tracker/minutesFairlyActive": [
-        "Tracker Minutes Fairly Active",
-        TIME_MINUTES,
-        "walk",
-    ],
-    "activities/tracker/minutesLightlyActive": [
-        "Tracker Minutes Lightly Active",
-        TIME_MINUTES,
-        "walk",
-    ],
-    "activities/tracker/minutesSedentary": [
-        "Tracker Minutes Sedentary",
-        TIME_MINUTES,
-        "seat-recline-normal",
-    ],
-    "activities/tracker/minutesVeryActive": [
-        "Tracker Minutes Very Active",
-        TIME_MINUTES,
-        "run",
-    ],
-    "activities/tracker/steps": ["Tracker Steps", "steps", "walk"],
-    "body/bmi": ["BMI", "BMI", "human"],
-    "body/fat": ["Body Fat", PERCENTAGE, "human"],
-    "body/weight": ["Weight", "", "human"],
-    "devices/battery": ["Battery", None, None],
-    "sleep/awakeningsCount": ["Awakenings Count", "times awaken", "sleep"],
-    "sleep/efficiency": ["Sleep Efficiency", PERCENTAGE, "sleep"],
-    "sleep/minutesAfterWakeup": ["Minutes After Wakeup", TIME_MINUTES, "sleep"],
-    "sleep/minutesAsleep": ["Sleep Minutes Asleep", TIME_MINUTES, "sleep"],
-    "sleep/minutesAwake": ["Sleep Minutes Awake", TIME_MINUTES, "sleep"],
-    "sleep/minutesToFallAsleep": [
-        "Sleep Minutes to Fall Asleep",
-        TIME_MINUTES,
-        "sleep",
-    ],
-    "sleep/startTime": ["Sleep Start Time", None, "clock"],
-    "sleep/timeInBed": ["Sleep Time in Bed", TIME_MINUTES, "hotel"],
-}
-
-FITBIT_MEASUREMENTS = {
-    "en_US": {
-        "duration": TIME_MILLISECONDS,
-        "distance": "mi",
-        "elevation": LENGTH_FEET,
-        "height": "in",
-        "weight": "lbs",
-        "body": "in",
-        "liquids": "fl. oz.",
-        "blood glucose": f"{MASS_MILLIGRAMS}/dL",
-        "battery": "",
-    },
-    "en_GB": {
-        "duration": TIME_MILLISECONDS,
-        "distance": "kilometers",
-        "elevation": "meters",
-        "height": "centimeters",
-        "weight": "stone",
-        "body": "centimeters",
-        "liquids": "milliliters",
-        "blood glucose": "mmol/L",
-        "battery": "",
-    },
-    "metric": {
-        "duration": TIME_MILLISECONDS,
-        "distance": "kilometers",
-        "elevation": "meters",
-        "height": "centimeters",
-        "weight": MASS_KILOGRAMS,
-        "body": "centimeters",
-        "liquids": "milliliters",
-        "blood glucose": "mmol/L",
-        "battery": "",
-    },
-}
-
-BATTERY_LEVELS = {"High": 100, "Medium": 50, "Low": 20, "Empty": 0}
-
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+from .const import (  # noqa: F401
```

Comment:
Is something re-exported here?

Reply:
yeah, it is because of `SCAN_INTERVAL`

Reply:
Where is it used?

Reply:
yeah it is similar to `PLATFORM_SCHEMA` (see [here](https://developers.home-assistant.io/docs/integration_fetching_data/#separate-polling-for-each-individual-entity)) and sure you are right - moved it back to platform file